File: AndroidJava_Hahow/BMI/app/src/main/java/com/codingbydumbbell/app_bmi/ResultActivity.java
package com.codingbydumbbell.app_bmi;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.widget.TextView;

public class ResultActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_result);

        TextView result = findViewById(R.id.result);
        float bmi = getIntent().getFloatExtra("BMI", 0);
        result.setText(getString(R.string.your_bmi_is) + bmi);
    }
}

File: AndroidJava_Hahow/BMI_Kotlin/app/src/main/java/com/codingbydumbbell/app_bmik/ResultActivity.kt
package com.codingbydumbbell.app_bmik

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_result.*

class ResultActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_result)
        val bmi = intent.getFloatExtra("BMI", 0f);
        result.text = "Your BMI is $bmi"
    }
}
